core/src/com/unciv/models/ruleset/Ruleset.kt

```diff
@@ -982,7 +982,7 @@ object RulesetCache : HashMap<String,Ruleset>() {
             // This happens if a building is dependent on a tech not in the base ruleset
             //  because newRuleset.updateBuildingCosts() in getComplexRuleset() throws an error
             Ruleset.RulesetErrorList()
-                .apply { add(ex.message!!.tr(), Ruleset.RulesetErrorSeverity.Error) }
+                .apply { add(ex.message, Ruleset.RulesetErrorSeverity.Error) }
```

Comment:
Most stuff going into RulesetErrorList has no translation handling yet, so if we want that long "Hey modder" list translated it will need groundwork, and that would likely translate on the other end, so this would fit in